### DISCUSSION TITLE
Non-linear brightness increase

### DISCUSSION THREAD

**Discussion by wilczarzgit** [MonitorControl/MonitorControl#1464] [General]
Hi, 

I use MonitorControl with LG 4K ULTRAFINE. It does the job done - thank you!

However, I have a few questions:

1) The external monitor seems to reach full brightness at about 60% of the range. How can I make it linear?

2) The brightness control only affects my external display. I did check the third "tick" in the general settings, but the macbook screen remains unaffected. Why is that?

3) Every once in a while the entire monitor displays articats. It's impossible to see anything. I can provide video if you need but it's really crazy! At first I thought my monitor was broken, but the restart brings it back. How can I avoid this?

Thanks!

**Comment by waydabber** [comment]
1. it's porbably 50%, this means DDC backlight control is not working, only software.
2. I am not sure
3. probably DDC is enabled and you are using HDMI which is not supported by MonitorControl and causes connection issues on some systems. Recommend using [BetterDisplay](https://betterdisplay.pro) instead for full support.
